frontend/src/summary.pug

```diff
@@ -127,12 +127,19 @@
             v-bind:href="getRepoLink(repo[0])", target="_blank",
             title="click to view group's repo"
           )
+            i.icon-button.fab.fa-github
```

Comment:
note that this line is added to fix the regression from #1106, which is missing the github icon when merge group and filterGroupSelection is 'groupByRepos'